Sources/ViewController+Swinject.swift

```diff
@@ -14,12 +14,12 @@ private var uivcRegistrationNameKey: String = "UIViewController.swinjectRegistra
 private var uivcWasInjectedKey: String = "UIViewController.wasInjected"
 
 extension UIViewController: RegistrationNameAssociatable, InjectionVerifiable {
-    internal var swinjectRegistrationName: String? {
+    public var swinjectRegistrationName: String? {
         get { return getAssociatedString(key: &uivcRegistrationNameKey) }
         set { setAssociatedString(newValue, key: &uivcRegistrationNameKey) }
     }
 
-    internal var wasInjected: Bool {
+    public var wasInjected: Bool {
```

Comment:
It's not sure we really need to make the properties public. We might revert it to internal later.